Raspi/Test/enum_topics.py

```diff
@@ -0,0 +1,32 @@
+"""MQTT Topics for module data recording"""
+from enum import Enum, unique
+
+
+@unique
+class NeatEnum(Enum):
+    """Our base Enum class"""
+
+    def __str__(self):
+        return self.value
+
+
+class WirelessModule(NeatEnum):
+    """Building blocks to make a topic for a wireless module. All that is
+    needed is to add a <module_id> number."""
+
+    base = '/v3/wireless-module/'
+
+    def battery(module_id):
+        return '/v3/wireless-module/' + str(module_id) + '/battery'
```

Comment:
Did you mean to reference `base` here rather than hardcode the string? Could also use an fstring.